core/test/common/Testable.js

```diff
@@ -23,8 +23,8 @@ contract("Testable", function() {
   });
 
   it("In test environment, different Testable contracts reference the same Timer", async function() {
-    const testable1 = await TestableTest.new(Timer.address);
-    const testable2 = await TestableTest.new(Timer.address);
+    const testable1 = await TestableTest.new((await Timer.deployed()).address);
+    const testable2 = await TestableTest.new((await Timer.deployed()).address);
```

Comment:
Wouldn't these be the same address? You can just `await` once and then pass in the address.

```suggestion
    const timer = await Timer.deployed();
    const testable1 = await TestableTest.new(timer.address);
    const testable2 = await TestableTest.new(timer.address);
```

Reply:
yup. I made a similar comment below :)